fvm/evm/debug/uploader.go

```diff
@@ -0,0 +1,61 @@
+package debug
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"cloud.google.com/go/storage"
+)
+
+type Uploader interface {
+	Upload(id string, data json.RawMessage) error
+}
+
+var _ Uploader = &GCPUploader{}
+
+type GCPUploader struct {
+	client *storage.Client
```

Comment:
Is this really not used?

Reply:
it's not, will do a bit more of cleanup and check the tests now